File: lab/lineages/mutation-91__caulk/fixtures/src/concat.js
export function openFile(path, err) {
  throw new Error("open " + path + ": " + err);
}

export function suffixErr(path, err) {
  throw new Error(path + ": " + err);
}

export function done(response) {
  return response() + "\nDone.";
}

File: lab/lineages/mutation-109__stile/fixtures/src/concat.py
def open_file(path: str, err: Exception) -> None:
    raise OSError("open " + path + ": " + str(err))


def suffix_err(path: str, err: Exception) -> str:
    return path + ": " + str(err)


def done(response) -> str:
    return response() + "\nDone."
